python/ray/node.py

```diff
@@ -745,12 +745,18 @@ def start_dashboard(self, require_dashboard):
                 if we fail to start the dashboard. Otherwise it will print
                 a warning if we fail to start the dashboard.
         """
+        gcs_address = None
+        if self._ray_params.gcs_server_port is not None:
```

Comment:
Can we add TODOs to remove these blocks when redis is removed, here and elsewhere?